src/reducers/shell.js

```diff
@@ -201,6 +203,12 @@ export default (state: Object = initialState, action: Object) => {
         smartContainerOpen: action.open
       };
 
+    case types.UPDATE_OVERLAY:
```

Comment:
can we move it to overlay reducer as this is overlay action?

Reply:
we can but it changes the shell prop, it seems like it better to handle all shell in one component